tools/parseschema2.py

```diff
@@ -271,18 +275,18 @@ def geteldocs(self, ename):
             os.mkdir(output_directory)
             ms.create(schema, output_directory)
 
-    if "java" in args.lang:
-        import langs.java as java
-        output_directory = os.path.join(outdir, "java")
-        if os.path.exists(output_directory):
-            lg.debug("Removing old Java output directory")
-            shutil.rmtree(output_directory)
-        os.mkdir(output_directory)
-        java.create(schema, output_directory)
-        if args.includes:
-            java.parse_includes(output_directory, args.includes)
-
-    if "csharp" in l_langs.lower():
+        if "java" in l_langs.lower():
```

Comment:
Is this indentation itended?

Reply:
Yes, in order to treat every language the same.